sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/splittabledofn/GrowableOffsetRangeTrackerTest.java

```diff
@@ -183,7 +183,7 @@ public void testProgressAfterFinished() throws Exception {
     tracker.checkDone();
     simpleEstimator.setEstimateRangeEnd(0L);
     Progress currentProgress = tracker.getProgress();
-    assertEquals(Long.MAX_VALUE - 10L, currentProgress.getWorkCompleted(), 0.001);
+    assertEquals(Long.MAX_VALUE - 10L, currentProgress.getWorkCompleted(), 0);
```

Comment:
I think the spirit of https://errorprone.info/bugpattern/FloatingPointAssertionWithinEpsilon is that we shouldn't specify epsilon, since it's small enough that we're doing an exact equality check anyway.
```suggestion
    assertEquals(Long.MAX_VALUE - 10L, currentProgress.getWorkCompleted();
```

Reply:
When I remove delta here, the assertion fails with: `Use assertEquals(expected, actual, delta) to compare floating-point numbers`

Reply:
Ha, caught between a rock and a hard place :) I think using epsilon=0 is a fine solution then.